ReactAndroid/src/main/java/com/facebook/react/modules/image/ImageLoaderModule.java

```diff
@@ -78,6 +85,9 @@ public void getSize(
       return;
     }
 
+    // Ensure FrescoHelpers.initialize was called.
+    this.initialize();
```

Comment:
I add some breakpoints in Android Studio, launch UIExplorer app, and found `ImageLoader.prefetch` was called before `ImageLoader.initialize`. I haven't found out the root cause, but i think it's worth send another PR.